crypto/fipsmodule/aes/asm/aesni-xts-avx512.pl

```diff
@@ -1460,7 +1460,10 @@
 
   my $rndsuffix = &random_string();
 
-  $code .= ".text\n";
+  $code .= <<___;
```

Comment:
It's only needed so that the pre-generated assembly code is usable with GCC 4.8.x: https://github.com/aws/aws-lc/pull/1665/files#diff-f797f62cb82976f63cec2e5936234848fab2e19af04010edc1945c822d291cf0R7

The `aesni-gcm-avx512` assembly file is unconditionally included in the build for x86-64:
https://github.com/aws/aws-lc/blob/bf992e705fa475abcff9e9f2a137babbd0399164/crypto/fipsmodule/CMakeLists.txt#L30